packages/example-app-base/src/components/Highlight/Highlight.tsx

```diff
@@ -1,9 +1,9 @@
 import * as React from 'react';
-import * as hljs from 'highlight.js/lib/highlight';
+import { registerLanguage } from 'highlight.js';
 import * as javascript from 'highlight.js/lib/languages/javascript';
```

Comment:
This import here is preventing us from turning on noImplicitAny. Not familiar enough with highlight.js to figure out if there is another way to do the initialization using stuff in the @types.